.gitignore

```diff
@@ -111,3 +111,20 @@ configure*
 *.bak
 Thumbs.db
 .directory
+*.db
```

Comment:
These changes in .gitignore are not needed. Looks like this is caused because of specific project configuration.